amplify/backend/function/expressServer/src/controllers/portfolioController.js

```diff
@@ -1,30 +1,23 @@
 const User = require('../models/user.model')
-const Portfolio = require('../models/portfolio.model')
+const Portfolio = require('../models/portfolio.model');
+const PortfolioService = require('../services/portfolioServices');
 
 
 
-// @desc create new portfolio
+// @desc create new hanging portfolio
 // route post portfolio/createPortfolio
 // @access private
 
-const createPortfolio = async (req, res, next) => {
+const createHangingPortfolio = async (req, res, next) => {
   try {
-      dateCreated = new Date()
-      if (!req.body.remainder){
-        remainder = 10000
-      }
-      else {
-        remainder = req.body.remainder
-      }
-      const newPortfolio = new Portfolio({
+    // create object to be used in createPortfolio service
+      const portfolioData = {
         portfolioName: req.body.portfolioName,
-        remainder: remainder,
-        dateCreated: dateCreated
-      });
-      await User.updateOne({ _id: req.user.id }, {$push: {portfolios: newPortfolio}})
-      await newPortfolio.save()
-        
-      res.json({ newPortfolio });
+        startingBalance: req.body.startingBalance,
+        userId: req.user.id
+      }
+      const createdPortfolio = PortfolioService.createPortfolio(portfolioData)
+      res.json(createdPortfolio);
```

Comment:
This will always be empty, update in excel too shouldn't be empty should be the portfolio object.